app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java

```diff
@@ -235,6 +278,20 @@ private void setUploadProgress( int uploadCount){
                 (uploadCount +"/" + levelInfo.getMaxUploadCount() );
     }
 
+    /**
+     * used to set the non revert image percentage
+     * @param notRevertPercentage
+     */
+    private void setImageRevertPercentage( int notRevertPercentage){
```

Comment:
remove whitespace. :)

Reply:
I can't find white spaces here but throughout the code, I tried to remove all extra white spaces :)

Reply:
I think he means the extra whitespace between `(` and `int` ;)

In general it's a good idea to follow the style of existing code. For instance, observe the places there are whitespaces (and no whitespaces) here:
`public void onCreate(Bundle savedInstanceState) {`